chemprop/featurizers/atom.py

```diff
@@ -36,44 +36,51 @@ class MultiHotAtomFeaturizer(AtomFeaturizer):
     +---------------------+-----------------+--------------+
     | slice [start, stop) | subfeature      | unknown pad? |
```

Comment:
```suggestion
    | slice [start, stop) | subfeature      | pad for unknown? |
```
I just realized that "unknown pad" doesn't mean that we don't know how big the padding is, but that it means there is a (single) padding bit set aside for any values not explicitly in the subfeature. That is probably obvious for others, but I've been confused about this for a while. Perhaps "pad for unknown" is more clear. If you made this change, then all the following rows would also need to have this column made wider.